projects/storefrontlib/src/cms-components/cart/cart-shared/cart-item-list/cart-item-list.component.ts

```diff
@@ -55,7 +55,8 @@ export class CartItemListComponent {
 
   constructor(
     protected activeCartService: ActiveCartService,
-    protected selectiveCartService: SelectiveCartService
+    protected selectiveCartService: SelectiveCartService,
+    public featureConfigService?: FeatureConfigService
```

Comment:
Would you mind creating a ticket to clean up the feature-level conditions introduced in this PR?

Reply:
Clean up ticket created: https://github.com/SAP/spartacus/issues/11037